src/components/simulation/SimulationPanel.svelte

```diff
@@ -117,7 +118,7 @@
 
   $: modelParametersMap = $plan?.model?.parameters?.parameters ?? {};
   $: if ($simulation && $plan) {
-    effects.getEffectiveModelArguments($plan.model.id, $simulation.arguments, user).then(response => {
+    effects.getEffectiveModelArguments($plan.model.id, {}, user).then(response => {
```

Comment:
Can you clarify (maybe in code comment) what this 2nd arg of `getEffectiveModelArguments` represents? Not sure I understand why it needed to be passed before but can be `{}` now

Reply:
@dandelany It's essentially a way to provide the overridden arguments to the function so that the response contains a full set of arguments determined by the model with the overridden arguments included in the response. With the `$simulation.arguments` being provided, the response wasn't truly returning "defaultArguments".